### PR TITLE
Skip

### PR DESCRIPTION
**Pull request by lokal-profil**
Use the Inaktiv column to skip creation of certain projects

### CODE REVIEW

**Comment by lokal-profil** [comment]
Should be rebased on #14

**Comment by lokal-profil** [comment]
Adding this since there are now 4 such rows for 2020

**Comment by lokal-profil** [comment]
> Should be rebased on #14

done

**Review by sebastian-berlin-wmse — DISMISSED** [review]
Implementation looks good and works. I'd just like a log message when a skipping a project explaining why.

File: project_start.py [hunk 1]
```diff
@@ -221,6 +221,9 @@ def add_phab_project(project_information, project_columns):
     with open(args.project_file[0]) as file_:
         projects_reader = csv.DictReader(file_, delimiter="\t")
         for project_information in projects_reader:
+            if project_information[project_columns["skip"]]:
+                # projects marked as inactive/skip should not be processed
```
Comment (lokal-profil) [inline project_start.py]:
```suggestion
                logging.info("Skipping '{}', marked as inactive.".format(project_information[project_columns["english_name"]]))
```

Comment (lokal-profil) [reply project_start.py]: @sebastian-berlin-wmse Would something like this work?

**Review by sebastian-berlin-wmse — APPROVED** [review]

**Pull request merged**